.github/workflows/deploy.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           deploy-message: "Deploy from GitHub Actions"
           enable-pull-request-comment: true
           enable-commit-comment: false
-          overwrites-pull-request-comment: true
+          overwrites-pull-request-comment: false
```

Comment:
So instead of having every deploy post its own comment, we might instead try a different strategy. In the `nwtgck/actions-netlify` readme, this is listed as an option: `deploy-message A custom deploy message to see on Netlify deployment (e.g. ${{ github.event.pull_request.title }})`

If we simply set one of the deploys with a custom message including the URLs for all the expected deploys, then we should be able to use the "overwrite comment" option. If we put the comment on the `www` build, the `course` and `course-v2` deploys might not be ready yet when the comment is posted / updated, but I think that's a fine tradeoff and better than the PR getting flooded with comments. Since the PR number will be the same for each deploy, we should have all the info we need.